Jenkinsfile

```diff
@@ -12,7 +11,7 @@ def rocmtestnode(Map conf) {
     def docker_args = conf.get("docker_args", "")
     def docker_build_args = conf.get("docker_build_args", "")
     def pre = conf.get("pre", {})
-    def ccache = "/var/jenkins/.cache/ccache"
+    def ccache = "/home/jenkins/workspace/.cache/ccache"
```

Comment:
It seems it would be better to use `/home/jenkins/ccache`. I assume the workspace directory is managed by jenkins so I dont know if there is a conflict.